master/getting-started/kubernetes/installation/gce.md

```diff
@@ -56,15 +56,12 @@ You can verify the rule with this command:
 gcloud compute firewall-rules list
 ```
 
-### 1.3 Clone this project
+### 1.3 Download the required files
 
-Clone the project
-
-    git clone https://github.com/projectcalico/calico.git
-
-And change into the directory for this guide.
-
-    cd calico/{{page.version}}/getting-started/kubernetes/installation
+    mkdir cloud-config; cd cloud-config
+    curl -O {{site.url}}{{page.dir | replace: 'installation', 'cloud-config' }}master-config.yaml
+    curl -O {{site.url}}{{page.dir | replace: 'installation', 'cloud-config' }}node-config.yaml
```

Comment:
I'd prefer we explicitly print the full URL to `*-config.yaml` instead of this method, as this method asserts that `gce.md` stays in `installation/`, and that `cloud-config` is a sibling folder with the yamls.

I could imagine it be easy for a maintainer to break one of those conditions and not notice this link is then wrong if it's written this way, as opposed to if it was just a full `{{site.baseurl}}{{page.version}}/getting-started/kubernetes/cloud-config/...`